### ISSUE TITLE
A11y_Visual Studio Code Services_Docs User Guide_Emmet_ScreenReader:Narrator is not announcing "column header"

### ISSUE THREAD

**Issue by divyabharadwaj92** [microsoft/vscode-docs#5098]
**GitHubTags**

#A11yMAS; #A11yTCS; #A11ySev2; #DesktopWeb; #Visual Studio Code Services; #BM-Visual Studio Code Services-Web-Jan2022; #Web; #Narrator; #WCAG1.3.1; #FTP; #STP; #ChromiumEdge;

**Environment Details**
Application Name: Visual Studio Code Services
Windows Version: Windows 11

**Repro Steps**

1. Launch the application : visual studio code services
2. Navigate to "Docs" link.
3. From left navigation, select "User Guide".
4. Navigate to "Emmet" link.
5. Observed that Narrator is not announcing the column header.

**Actual:**
The narrator is not announcing the "column header".
NOTE: This issue is not reproducing in NVDA screenreader.

**Expected Result**
Narrator should announce the "column header"

**User Impact**
Screen reader users may not understand the content clearly if "Column header" is not specified.

**Narrator Attachment**
![A11y_Visual Studio Code Services_Docs User Guide_Emmet_Screen Reader](https://user-images.githubusercontent.com/97668149/149756122-33602d50-27bf-4347-ae67-aebe0d9186d1.gif)

**Comment by rzhao271** [comment]
I noticed in the screenshot above, the words "Column header" do show up, but they show up late. I also looked at the code for that table, and do not see anything unusual. The ARIA role there should implicitly be `columnheader`.

I wonder if it is a Narrator issue, rather than a site issue?

![Screenshot showing how in the demo GIF above, the words column header do show up](https://user-images.githubusercontent.com/7199958/153076205-9f4625e9-7864-483c-b541-b6a498e7acdd.PNG)

**Comment by rzhao271** [comment]
Also, the HTML for that table already contains `thead`, `th` elements and so on, so the table has the correct semantic tags and implicit ARIA labels.
I'm not sure what else can be done here.

**Comment by fsteffi** [comment]
no issue https://code.visualstudio.com/docs/editor/emmet

**GithubTags:**#Closed;
[Uploading 5098-closed.webm…]()
